### PR TITLE
Add nav bar icons, change footer icons to svg

### PR DESCRIPTION
**Pull request by Ne-anna**
Added necessary items to navigation bar - cart, cart-counter, divider, burger menu icon. 
Change color in footer and change social media icons to svg format icons.

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -18,6 +18,12 @@
             src="/images/cart-shopping-solid.svg"
             alt="cart-shopping-solid"
           />
+          <div>
```
Comment (ReinisZogots) [inline Index.html]: No need to wrap them in div

File: style.css [hunk 2]
```diff
@@ -237,6 +248,7 @@ footer {
 
 .footer-logo-image {
   align-items: center;
+  width: 10%;
```
Comment (ReinisZogots) [inline style.css:249]: Do you need the logo to stretch?

Comment (Ne-anna) [reply style.css:249]: Yes, because if I remove this width:10% the icons become to large to fit in one line.

File: Index.html [hunk 3]
```diff
@@ -18,6 +18,12 @@
...
+            <P class="navigation-count">2</P>
+          </div>
+          <div>
+            <p class="navigation-divide">|</p>
```
Comment (EgijaLocmele) [inline Index.html]: Don't add it like this, use icon

File: style.css [hunk 4]
```diff
@@ -53,15 +54,25 @@ body {
...
 }
 
 .navigation-right-cart {
-  width: 25px;
+  width: 18px;
+}
+
+.navigation-count {
+  font-size: 15px;
+  font-weight: bold;
```
Comment (EgijaLocmele) [inline style.css:68]: Why did you decide to use `bold ` here and not `700 ` as before? Better to use just one method

File: style.css [hunk 5]
```diff
@@ -109,8 +120,8 @@ body {
 button {
   background-color: transparent;
   border-color: var(--white);
-  margin-top: 55px;
-  margin-left: 15px;
+  margin-top: 50px;
```
Comment (EgijaLocmele) [inline style.css:123]: Fyi, you can shorten it to just one line -  `margin: 50px 0 0 22px;`
Not necessary to do it like that, but just keep in mind that it's possible like that also :)

**Review by ReinisZogots — APPROVED** [review]

File: style.css (hunk 5 above)
Comment (Ne-anna) [reply style.css:123]: Thank you for reminder, but for now I'm gonna stick to separate margin writing, just to get  used to it, hope that soon I'm gonna start to use your suggested method. :)

File: Index.html (hunk 3 above)
Comment (Ne-anna) [reply Index.html]: Fixed

**Review by EgijaLocmele — APPROVED** [review]

**Pull request merged**